.github/workflows/test.yml

```diff
@@ -1330,4 +1421,4 @@ jobs:
         if: always()
         with:
           name: ${{ github.job }}_${{ matrix.os }}_node-${{ matrix.node }}_${{ matrix.queryEngine }}_junit.xml
-          path: packages/*/junit.xml
+          path: packages/*/junit*.xml
```

Comment:
Why the additional `*`? Does `JEST_JUNIT_UNIQUE_OUTPUT_NAME: true` influence this?

Reply:
Yes, this is to make sure we match the file that now contains an id

Reply:
Then this should actually work - seems Buildpulse only uses the string inside to match the test, not the filename as I thought. Nice!